ArmoniK.Extension.CSharp.Client/Common/Services/IBlobService.cs

```diff
@@ -151,6 +132,7 @@ public static class BlobServiceExt
   /// <param name="quantity">The number of blobs to create metadata for.</param>
   /// <param name="cancellationToken">A token to monitor for cancellation requests.</param>
   /// <returns>An asynchronous enumerable of blob information objects.</returns>
+  [PublicAPI]
```

Comment:
Either you don't put them or you put them every where

Reply:
It should be done in another PR if you need to put it on every method from every interface